latex_table.py

```diff
@@ -17,7 +17,7 @@
     "HippyVM": "HippyVM",
     "PyHyp-mono": "PyHyp$_m$",
     "PyHyp-comp": "PyHyp$_c$",
-    "PyHyp-comp-rev": "PyHyp$_{c'}$",
+    "PyHyp-comp-rev": "PyHyp$_{c2}$",
```

Comment:
Ah, yes, I remember. We could maybe call it "inverse" or something? Or maybe `c'` is as good as it gets...